_posts/jobs/2016-09-30-android.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Senior Android Engineer"
+description: We want you!
+modified:
+categories: jobs
+author: peter_krauss
+excerpt: We are looking for a Senior Android Engineer.
+tags: [Jobs, Android, Senior]
+date: 2016-09-30T10:39:55-04:00
+---
+## About the Role
+You are a Software Engineer by heart. You have clean code for breakfast sprinkled with MVVM or MVP, then you inject your dependencies while driving reactively to work. Then you integrate continuously and release it by lunchtime. In the afternoon you get up-to-date with the latest technologies and before leaving you improve the stress testing, add a espresso test or do whatever is necessary in order to have a monkey runner proof system you are proud of. You go to bed with the DoD under your pillow and dream of unit tests with a smile on your face.
+
+**You are:**
+
+* a person we could learn from.
+* interested in both, your quality of code and creating a great product
+* seeing problems yourself and take ownership solving them
+* striving to develop yourself and your team
+* cool to be around
+
+**Requirements (You have gathered professional experiences in):**
+
+* Android Core, RxJava and Dependency Injection
+* Gradle (Groovy)
+* Persistency Technologies (SQLite, Content Providers)
+* Test Driven Development (TDD - unit and UI)
+* Monitoring and maintaining the app (Fabric, Charles and Memory Profiling)
+* Continuous Integration/Continuous Delivery
+
+**Non functional Requirements**
+* BA/BS in Computer Science or equivalent practical experience.
```

Comment:
Let's not punctuate the [bullet points] (http://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html).